content/fr/docs/concepts/services-networking/ingress.md

```diff
@@ -328,7 +328,7 @@ metadata:
 type: kubernetes.io/tls
 ```
 
-Référencer ce secret dans un Ingress indiquera au contrôleur d'ingress de sécuriser le canal du client au load-balancer à l'aide de TLS. Vous devez vous assurer que le secret TLS que vous avez créé provenait d'un certificat contenant un CN pour `sslexample.foo.com`.
+Référencer ce secret dans un Ingress indiquera au contrôleur d'ingress de sécuriser le canal du client au load-balancer à l'aide de TLS. Vous devez vous assurer que le secret TLS que vous avez créé provenait d'un certificat contenant un CN pour `tlsexample.foo.com`.
 
 ```yaml
```

Comment:
remplace all yaml block with this code like the EN page : 
{{< codenew file="service/networking/tls-example-ingress.yaml" >}}

Reply:
Done

Reply:
je voulais dire de remplacer tout le code du yaml par {{< codenew file="service/networking/tls-example-ingress.yaml" >}}

cela permet de toujours pointer sur l'exemple de référence.